File: pset1/greedy.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    
    float money;
    int money_coins;
    int count=0;
   
   do
   {
    printf("How much change is owed?\n");
    money = GetFloat();
   }while(money < 0);
  
    money_coins = round(money*100);
   
    while(money_coins>=25)      
    {
        money_coins -= 25; 
        count++;
    }
    while(money_coins>=10)
    {   
        money_coins -= 10;
        count++;
    }
    while(money_coins>=5)
    {
        money_coins -= 5;
        count++;
    }
        
    while(money_coins>=1)
    {
        money_coins -= 1;
        count++;
    }
       
    printf("%d\n",count);
        
    
    return 0;
}

File: pset6/Python_programs/greedy.py
from cs50 import get_float

# cents: 25, 10, 5 , 1

while True:
    print('How much change is owed?')
    n= get_float()
    if n>=0:
        break

money_coins = round(n*100)
count = 0

while money_coins>=25:
    money_coins -= 25
    count+=1;

while money_coins>=10:
    money_coins -= 10
    count+=1;

while money_coins>=5:
    money_coins -= 5
    count+=1;

while money_coins>=1:
    money_coins -= 1
    count+=1;

print('%d' %count)
        
